src/Nethermind/Nethermind.Serialization.SszGenerator/Attrs.cs

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+namespace Nethermind.Serialization.Ssz;
+
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct)]
+public class SszSerializableAttribute : Attribute;
```

Comment:
why do we have duplicate in `[SszSerializableAttribute.cs](https://github.com/NethermindEth/nethermind/pull/7299/files#diff-6690621b54f17ca1def35ba5fbac08468830ce2d8fa917afaf4a0cac67d61195)`?